.github/workflows/ci.yml

```diff
@@ -31,22 +42,33 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        shard_file:
+          - shard.yml
         crystal_version:
           - 0.36.1
           - 1.0.0
         experimental:
           - false
+        include:
+          - shard_file: shard.edge.yml
+            crystal_version: 1.0.0
+            experimental: true
```

Comment:
why is 1.0.0 experimental? Wouldn't this be tested from line 49 above?

Reply:
It's experimental because it's using the `shard.edge.yml`! Every `includes` just has to have every matrix value, and I figured we'd want to test the all-`master` shard file against the latest Crystal version we support.